tools/doc/html.js

```diff
@@ -152,6 +158,16 @@ function parseLists(input) {
   return output;
 }
 
+function parseYAML(text) {
+  const meta = common.extractAndParseYAML(text);
+  let html = '<div class="api_metadata">';
+
+  Object.keys(meta).forEach(key => {
+    html += '<span>' + key + ': ' + meta[key] + '</span>';
+  });
```

Comment:
Could we specifically add items (like `added`) from the YAML instead of appending all of them, so we don't block purely informational items from being added?